packages/kolibri/components/pages/AppBarPage/index.vue

```diff
@@ -3,6 +3,13 @@
   <!-- TODO useScrollPosition to set scrollPosition...
     here or in router, but somewhere -->
   <div class="main">
+    <div
+      v-if="windowIsSmall"
+      ref="swipeZone"
+      :class="{ 'rtl-swipe-zone': isRtl.value }"
```

Comment:
You shouldn't need to reference the `.value` attribute in a template, `isRtl` should suffice.